cohort-engine/src/main/java/com/ibm/cohort/engine/measure/MeasureEvaluator.java

```diff
@@ -205,13 +208,27 @@ public MeasureReport evaluatePatientMeasure(Measure measure, String patientId, S
 		return evaluation.evaluatePatientMeasure(measure, seed.getContext(), patientId);
 	}
 
+	private void setDefaultValue(Context context, ParameterDefinition parameterDefinition) {
+		parameterDefinition.getExtension().stream()
+				.filter(MeasureEvaluator::isDefaultValue)
+				.forEach(defaultValue ->
+						         context.setParameter(
+						         		null,
```

Comment:
I know this is just refactored code as part of this PR, but I wanted to confirm that `null` here makes sure that the parameter is global/not tied to a specific library.